homeassistant/components/deconz/__init__.py

```diff
@@ -122,9 +130,19 @@ def async_configure(call):
         http://dresden-elektronik.github.io/deconz-rest-doc/rest/
         """
         deconz = hass.data[DOMAIN]
+        entities = hass.data[DECONZ_ENTITIES]
         field = call.data.get(SERVICE_FIELD)
-        data = call.data.get(SERVICE_DATA)
-        yield from deconz.async_put_state(field, data)
+        entity_id = call.data.get(SERVICE_ENTITY)
+        data_string = call.data.get(SERVICE_DATA)
+        data = ast.literal_eval(data_string)  # String to dict
```

Comment:
Eval is not allowed, too dangerous.

Reply:
Do you have suggestion about how I can get a dictionary here?

Reply:
The Issue here is that data is expected to be json compliant, which it unfortunately isn't since it get single quotes, where as json.loads expects double quotes. So there is a translation going on from the service entry in the gui to the service in the backend; {"name": "TRADFRI motion sensor 1"} arrives as {'name': 'TRADFRI motion sensor 1'}